ExampleMod/Items/Placeable/ExampleCommandSign.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Items.Placeable
+{
+	public class ExampleCommandSign : ModItem
+	{
+		public override void SetStaticDefaults()
+		{
+			Tooltip.SetDefault("This is a modded sign.");
```

Comment:
Maybe make it more descriptive, like: "Enter a valid command on this sign to execute it!"